flink-runtime-web/web-dashboard/src/app/pages/job-manager/log-detail/job-manager-log-detail.component.html

```diff
@@ -16,5 +16,21 @@
   ~ limitations under the License.
   -->
 
+<div class="breadcrumb">
+  <nz-breadcrumb>
+    <nz-breadcrumb-item>
+      <a [routerLink]="['../']"><i nz-icon type="rollback" theme="outline"></i> Log List</a>
+    </nz-breadcrumb-item>
+    <nz-breadcrumb-item>
+      {{ logName }}
+    </nz-breadcrumb-item>
+  </nz-breadcrumb>
+  <flink-refresh-download
+    [isLoading]="isLoading"
+    [downloadHref]="downloadUrl"
+    [downloadName]="logName"
+    (reload)="reload()"
+    (fullScreen)="toggleFullScreen($event)">
+  </flink-refresh-download>
+</div>
 <flink-monaco-editor [value]="logs"></flink-monaco-editor>
```

Comment:
When I scroll the log content, a border appears at the top of the editor, I think it would be nice to hide it, but it's also ok if we want to keep it :)

![image](https://user-images.githubusercontent.com/11643113/79228009-5d5eaf00-7e93-11ea-98a6-b59322922a91.png)

Reply:
fixed